web_ui/src/world.ts

```diff
@@ -147,5 +147,12 @@ export const Current: World = {
           `/v1/t/${args.teamId}/subscription_info`,
         )
         .then(d => d.data),
+    updateStripeCustomerInfo: (args: api.UpdateStripeCustomerInfoArgs) => {
+      const { teamId, ...payload } = args
+      return authRoute.post<unknown>(
+        `/v1/t/${args.teamId}/update_stripe_customer_info`,
+        payload,
```

Comment:
we're using JSON instead of formData. We had to move the api to `app.kodiakhq.com` to fix CORS issues with `application/json` Content Type.